core/src/js/components/accordion/accordion.js

```diff
@@ -0,0 +1,53 @@
+const accordions = document.querySelectorAll('.su-accordion');
+const titleButtons = document.querySelectorAll('.su-accordion__button');
+const expandButtons = document.querySelectorAll('.su-accordion__expand-all');
+const collapseButtons = document.querySelectorAll('.su-accordion__collapse-all');
+
+const isExpanded = x => x.getAttribute('aria-expanded') === 'true';
+const setExpanded = (x, value) => x.setAttribute('aria-expanded', value);
+const setHidden = (x, value) => x.setAttribute('aria-hidden', value);
+
+document.addEventListener('DOMContentLoaded', event => {
+  Array.prototype.forEach.call(accordions, acc => {
+    acc.classList.remove('no-js');
```

Comment:
Is the no-js needed? Do you have plans to add styles based on it?

Reply:
That's a good point. Let me think about this a bit more.